recipe/migrations/r400.yaml

```diff
@@ -18,6 +18,10 @@ __migrator:
     1
   automerge: true
 
+  # use azure since things are failing on travis (2020/05/01)
+  conda_forge_yml_patches:
+    provider.linux_ppc64le: azure
```

Comment:
Is this going to add azure for feedstocks that don't have ppc64le enabled?

Reply:
nope it should not.

Reply:
I wrote a test in the code to make sure.

Reply:
see here: https://github.com/regro/cf-scripts/blob/master/tests/test_patch_dict.py